data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -336,8 +336,64 @@ do
 	sleep 5
 done
 
-echo "Starting cluster-bootstrap..."
 
+{{if .SingleNode}}
+
+if [ ! -f cb-bootstrap.done ]
+then
+    bootkube_podman_run \
+        --rm \
+        --volume "$PWD:/assets:z" \
+        --volume /etc/kubernetes:/etc/kubernetes:z \
+        "${CLUSTER_BOOTSTRAP_IMAGE}" \
+        start --tear-down-early=false --asset-dir=/assets --bootstrap-in-place=true --required-pods=""
+        touch cb-bootstrap.done
+fi
+
+if [ ! -f update-master-ignition.done ]
+then
+    echo "Move etcd static pod"
+    mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
+
+    until ! crictl ps | grep etcd
+    do
+        echo "Waiting for etcd to go down"
+        sleep 10
+    done
+
+    echo "Getting master Ignition from MCS"
+    # Get the master ignition for MCS
+    curl -k  -H "Accept:'application/vnd.coreos.ignition+json;version=3.1.0, */*;q=0.1'" \
+    https://localhost:22623/config/master -o /opt/openshift/master.ign
+
+    echo "Updating master Ignition"
+    bootkube_podman_run \
+        --rm \
+        --privileged \
+        --volume "/var/lib/etcd:/var/lib/etcd" \
+        --volume "$PWD:/assets:z" \
+        --volume "/etc/kubernetes:/etc/kubernetes" \
+        "${CLUSTER_BOOTSTRAP_IMAGE}" \
+        bootstrap-in-place --asset-dir=/assets --ignition-path=/assets/master.ign
+
+    touch update-master-ignition.done
+fi
+
+if [ ! -f write-to-disk.done ]
+then
+    echo "Writing RHCOS + master Ignition to disk"
+    #TODO: which disk to use?
+    coreos-installer install --insecure -i /opt/openshift/master.ign /dev/vda
```

Comment:
I assume it should be parametrized? Env variable? Otherwise, how would we use it from assisted installer?

Reply:
To allow AssistedInstaller to pick up the flow, and be able to select disk, inject network customization, and other improvements, I assume we'll need some flag saying --bootstrap-only, meaning the bootstrap phase will finish with master.ign generated and the AI will continue from that point

Reply:
The AssistedInsatller can create these files to skip these operations:
/opt/openshift/write-to-disk.done
/opt/openshift/reboot.done